### PR TITLE
Add protobuf dependencies, also ignore .xz and ~ files.

### PR DESCRIPTION
**Pull request by jdb-vsn**
Also, add a README.

@dw-vsn @mb-vsn @pe-vsn @es-vsn

### CODE REVIEW

File: Godeps [hunk 1]
```diff
@@ -2,3 +2,5 @@ github.com/nsf/termbox-go bc836fdbcf75184c95c96500ea740e3dd727b5a8
 github.com/boltdb/bolt da700e8fdb065b7e073700c9afa8369d4c60c674
```
Comment (pe-vsn) [inline Godeps:2]: @dw-vsn Why is bolt still in here? I thought we decided against it for Canary? Or did you have it in mind for something else?

**Comment by pe-vsn** [comment]
General Go question: why again can't we at least exclude the pkg directory here?

**Comment by pe-vsn** [comment]
+1

**Comment by danw-volley** [comment]
> General Go question: why again can't we at least exclude the pkg directory here?

What do you mean "again", have we talked about this in the past? I don't remember...

**Comment by pe-vsn** [comment]
I wasn't sure if we had talked about this before (at the time of writing the comment: three weeks ago). But this issue has been addressed in my standing PR for this repo. If there is a debate about what is stored and what is not then that PR may be a more appropriate place to have that discussion because it deals with that directly.

File: Godeps (hunk 1 above)
Comment (danw-volley) [inline Godeps:2]: I use it in the Installer for the media systems to store SSH and Nacl keys.

**Comment by danw-volley** [comment]
+1, but given phil's changes, it sounds like you might need some tweaks.

**Comment by jdb-vsn** [comment]
ping @mb-vsn @es-vsn

**Comment by pe-vsn** [comment]
Joshua, it would not be appropriate to merge these yet until my [PR](https://github.com/vs-networks/go-ext/pull/4) is merged. At that point, you will want to rebase and run tools/update again. There are changes being made that make a lot of the files in here wrong to add.

**Comment by pe-vsn** [comment]
And I'm sorry for the wait; I'm just waiting for my gpm changes to be accepted upstream.

**Comment by pe-vsn** [comment]
@jdb-vsn, you are good to go now. Please rebase and remove your pkg directory. Feel free to run tools/update again but I don't think that is necessary anymore unless you explicitly want to maketar too.

**Comment by pe-vsn** [comment]
Since these dependencies were added in my last [commit](https://github.com/vs-networks/go-ext/commit/42e6cdc22b8e645b64c2a0df8d15d983afd67cd3), can we close this?

**Pull request closed without merging**